Sources/GRPC/_GRPCClientChannelHandler.swift

```diff
@@ -552,3 +586,13 @@ extension _GRPCClientChannelHandler: ChannelOutboundHandler {
     }
   }
 }
+
+// TODO: Extract type
+public struct HPACKHeadersInjector: InjectorProtocol {
+  public init() {}
+
+  public func inject(_ value: String, forKey key: String, into headers: inout HPACKHeaders) {
+    // TODO: Replace or add?
```

Comment:
hmm good question though I guess replace here.
technically the same header could be added a few times (is that also true in h2, i dont remember?, but i think for our purposes it's a set()